samples/bluetooth/st_ble_sensor/README.rst

```diff
@@ -0,0 +1,41 @@
+.. _bluetooth-st_ble_sensor:
+
+Bluetooth: ST BLE Sensor Demo
+#############################
+
+Overview
+********
+
+This application demonstrates BLE peripheral by exposing vendor-specific
+GATT services. Currently only button notification and LED service are
+implemented. Other BLE sensor services can easily be added.
+See `BlueST protocol`_ document for details of how to add a new service.
+
+Requirements
+************
+
+* `ST BLE Sensor Android app <ST BLE Sensor app>`_
+* A board with BLE support
+
+Building and Running
+********************
+
+This sample can be found under :zephyr_file:`samples/bluetooth/st_ble_sensor` in the
+Zephyr tree.
+
+See :ref:`bluetooth samples section <bluetooth-samples>` for details.
+
+.. _ST BLE Sensor app:
+    https://play.google.com/store/apps/details?id=com.st.bluems
+
+.. _ST Nucleo WB55RG:
```

Comment:
If you're not referencing these links in the document, it's best to delete them.

Reply:
fixed